server/prisma/migrations/20250123001634_library5/migration.sql

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Delete all 5 new migration directories and recreate so they are squashed into one.